File: Examples/Node.js/Files/MoveFile.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var StorageApi = require('asposestoragecloud');
var configProps = require('../config/config.json');
var data_path = '../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var outFolder = configProps.out_folder;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);

var name = 'testfile.txt';	
var dest = 'new-testfile.txt';

try {	
	storageApi.PutCreate(name, versionId=null, storage=null, file= data_path + name , function(responseMessage) {
		assert.equal(responseMessage.status, 'OK', '');
		storageApi.PostMoveFile(name, dest, versionId=null, storage=null, destStorage=null,function(responseMessage) {
			console.log('status:', responseMessage.status);
		});
	});    

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1

File: Examples/Perl/Files/MoveFile.pl
#ExStart:1
use lib 'lib';
use strict;
use warnings;
use utf8;
use File::Slurp; # From CPAN
use JSON;

use AsposeStorageCloud::StorageApi;
use AsposeStorageCloud::ApiClient;
use AsposeStorageCloud::Configuration;

my $configFile = '../Config/config.json';
my $configPropsText = read_file($configFile);
my $configProps = decode_json($configPropsText);
my $data_path = '../../../Data/';
my $out_path = $configProps->{'out_folder'};

$AsposeStorageCloud::Configuration::app_sid = $configProps->{'app_sid'};
$AsposeStorageCloud::Configuration::api_key = $configProps->{'api_key'};

# Instantiate Aspose.Storage Cloud API SDK
my $storageApi = AsposeStorageCloud::StorageApi->new();

# Invoke Aspose.Storage Cloud SDK API to move file      		                           
my $name = 'testfile.txt';	
my $dest = 'new-testfile.txt';
my $response = $storageApi->PutCreate(Path => $name, file => $data_path.$name);
$response = $storageApi->PostMoveFile(src => $name, dest=> $dest);
print $response->{'Status'};
#ExEnd:1